src/components/builds/BuildDetails.tsx

```diff
@@ -241,6 +242,10 @@ function BuildDetails(props: Props) {
       <Paper elevation={2}>
         <TaskList tasks={build.latestGroupTasks} />
       </Paper>
+      <div className={classes.gap} />
```

Comment:
Let's make sure to not show it at all if there are no hooks and let's move it before task list.

Reply:
Indeed, fixed in https://github.com/cirruslabs/cirrus-ci-web/pull/358/commits/ae109a2f9038c94569ee35bd75f444f16b943718.

Reply:
How about moving hook before tasks?

Reply:
I'd rather not do that because this will result in the "jumping UI", i.e. sometimes hooks are displayed and sometimes they are not, whereas the tasks are always displayed.

Reply:
My concern is that there can 100s of tasks and just a few hooks. If you have 100 tasks then you'll need to scroll for hooks.